### PR TITLE
NOISSUE - Fix paging

### PR DESCRIPTION
**Pull request by mijicd**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -124,13 +124,16 @@ Content-Length: 1105
...
 
 ```
-curl -s -S -i --cacert docker/ssl/certs/mainflux-server.crt --insecure -H "Authorization: <user_auth_token>" https://localhost/channels?offset=0&limit=5
+curl -s -S -i --cacert docker/ssl/certs/mainflux-server.crt --insecure -H "Authorization: <user_auth_token>" https://localhost/clients?offset=0&limit=5
 ```
-If you don't provide these params, default values will be used insted. Default value for
-offset is 0, and for limit is 10. Max value for limit is 100, and every value above that will be replaced by limit max value. Providing invalid values for these two params will be considered malformed request, so bad request will be received as response.
+
+If you don't provide them, default values will be used instead: 0 for `offset`,
+and 10. Note that `limit` cannot be set to values greater than 100. Providing
```
Comment (anovakovic01) [inline docs/getting-started.md]: Maybe to add for `limit` just to be clear?

File: docs/getting-started.md [hunk 2]
```diff
@@ -187,13 +190,16 @@ Content-Length: 139
...
+group of channels. In that case, your request should look like:
 
 ```
 curl -s -S -i --cacert docker/ssl/certs/mainflux-server.crt --insecure -H "Authorization: <user_auth_token>" https://localhost/channels?offset=0&limit=5
 ```
-If you don't provide these params, default values will be used insted. Default value for
-offset is 0, and for limit is 10. Max value for limit is 100, and every value above that will be replaced by limit max value. Providing invalid values for these two params will be considered malformed request, so bad request will be received as response.
+
+If you don't provide them, default values will be used instead: 0 for `offset`,
+and 10. Note that `limit` cannot be set to values greater than 100. Providing
```
Comment (anovakovic01) [inline docs/getting-started.md]: Maybe to add for `limit` just to be clear?

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

File: docs/getting-started.md (hunk 1 above)
Comment (drasko) [reply docs/getting-started.md]: Agreed with @anovakovic01

File: docs/getting-started.md (hunk 2 above)
Comment (drasko) [reply docs/getting-started.md]: +1

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Review by drasko — APPROVED** [review]

**Pull request merged**